src/Stratis.Bitcoin/Configuration/NodeSettings.cs

```diff
@@ -96,7 +96,8 @@ public class NodeSettings : IDisposable
             string agent = "StratisBitcoin", string[] args = null)
         {
             // Create the default logger factory and logger.
-            this.LoggerFactory = new ExtendedLoggerFactory();
+            var loggerFactory = new ExtendedLoggerFactory();
```

Comment:
I don't really get introduction of this local variable, but okay, but why is it not used everywhere then?

Reply:
LoggerFactory is `ILoggerFactory`, we need access to `ExendedLoggerFactory` to call the load method